### ISSUE TITLE
Capital Arcology Bug

### ISSUE THREAD

**Issue by WizOG** [Dukko/mergerofmods#23]
**Describe the bug**
Capital arcology unable to be built using the Single term goverment type.

**To Reproduce**
- Include Empire type/civics/etc if relevant
- Empire Type : Single term
- Select the "ap_arcology_project" AP
Then Try to create the Capital arcology on Empire capital.

**Expected behavior**
Due to the goverment type being democratic, I assumed i would be able to build the Capital Arcology on my empires capital planet. This might happen with the Democratic Autocracy aswell (Not checked).

 - Version 0.8.5

**Comment by Dukko** [comment]
Good catch. I need to include those two democratic governments in my patch. Thank you.
